runtime/jvm.xml.ftl

```diff
@@ -1282,7 +1282,15 @@
 									overwrite="false" verbose="true" />
 							</then>
 						</if>
-
+		
+						<if>
+							<not>
+								<available file="${r"${j9jcl.module.info}/module-info.java"}" type="file" />
+							</not>
+							<then>
+								<copy file="${r"${output}/${module.name}_extracted/classes/module-info.java"}" tofile="${r"${j9jcl.module.info}/module-info.java"}" /> 
+							</then>
```

Comment:
A `else` block can be followed to include following target `-merge-module-info`, otherwise it just merges two identical files unnecessarily.

Reply:
Pls add comments that the module-info might have been modified such that duplication cleaning up is necessary, also list the target modules by this change.